pkg/pillar/agentbase/apptracker.go

```diff
@@ -0,0 +1,556 @@
+// Copyright (c) 2025 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package agentbase
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+// TriState - the boolean type with 3 states for generic AppInfoItems
+type TriState int
+
+const (
+	// NotSet indicates the initial value, which is 0.
+	NotSet TriState = iota
+	// False indicates the false value.
+	False
+	// True indicates the true value.
+	True
+)
```

Comment:
Can't you use the existing types.TriState for this?

Reply:
Sure.